src/index.js

```diff
@@ -10,8 +10,10 @@ let app = new PIXI.Application({
   width: window.innerWidth,
   height: window.innerHeight,
   antialias: true,
-  resolution: 1
+  resolution: window.devicePixelRatio || 1,
+  autoResize: true
```

Comment:
Does https://github.com/chrismdp/tinyking/pull/5/files#diff-1fdf421c05c1140f6d71444ea2b27638R21 still need to be there given this line?

Reply:
I think the `antialias` line can now be removed, I'll quickly test it on a non-HiDPI screen to make sure.

Both lines added are required, the first gets the best resolution and then the autoResize scales it.

Reply:
Ah no - it's `app.renderer.autoResize = true;` on line 21 I wasn't sure about.

Reply:
Without the `autoResize` the resolution doubles and goes off the screen.

Reply:
Non-HiDPI Screenshot (1920x1080)
![Screenshot_20200712_192607](https://user-images.githubusercontent.com/294376/87253761-91874300-c475-11ea-945b-29b84d937f24.png)